### ISSUE TITLE
Deprecated command line argument `--src` leads to errors or missing proper documentation

### ISSUE THREAD

**Issue by kesmik** [jsh9/pydoclint#117]
Hi,

I am new to this awesome tool and I may not be entirely correct but for me it seems that `-s --src` are legacy options which do nothing or I could not find any documentation and could lead to wrong configuration. I have been happily used it for my code linting by expecting to get source files checked with it and just recently noticed that it does not catching errors.

Checked the entry code (in github only) and it seems it`s only checking that this option is not selected together with paths.
Did I missed something or this option can be removed?

If not maybe documentation is missing and should be added? Any explanation/help/guidelines would be appreciated.

**Comment by jsh9** [comment]
Hi @kesmik : I think you are right.  `--src` does nothing.

If you want to check a specific file, you can just do

```python
pydoclint a_specific_file.py
```

I going to remove `--src` then.

**Comment by kesmik** [comment]
Thanks!
